html/global_attributes.json

```diff
@@ -1131,22 +1131,9 @@
                 "version_removed": "23"
               }
             ],
-            "firefox_android": [
-              {
-                "version_added": "79"
-              },
-              {
-                "version_added": "68",
-                "version_removed": "79",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.forms.inputmode",
-                    "value_to_set": "true"
-                  }
-                ]
-              }
-            ],
+            "firefox_android": {
+              "version_added": "79"
```

Comment:
Do you know what explains the big difference to the desktop data above?

Reply:
It appears that this was added by #6782.  The [linked bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1631681) shows there was an intentional offset between enabling the feature on desktop and mobile.